appinfo/routes.php

```diff
@@ -14,5 +14,6 @@
         ['name' => 'config#setAdminConfig', 'url' => '/admin-config', 'verb' => 'PUT'],
         ['name' => 'config#getWidgetContent', 'url' => '/widget-content', 'verb' => 'GET'],
 		['name' => 'config#getWidgetImage', 'url' => '/widget/image/{fileId}', 'verb' => 'GET'],
+		['name' => 'config#enableWidget', 'url' => '/enable-widget', 'verb' => 'PUT'],
```

Comment:
`enable-widget` might be named as `default-enable-widget` or similarly to be more precise of the function of the route

Reply:
It does not change the default dashboard layout but just enables the welcome widget so I think the route name is fine. WDYT?